.circleci/real_config.yml

```diff
@@ -930,6 +936,14 @@ commands:
                 command: |
                   echo 'export HELM_VALUES="${HELM_VALUES},taskContainerDefaults.cpuImage=<<parameters.environment-image>>,taskContainerDefaults.gpuImage=<<parameters.environment-image>>"' >> "$BASH_ENV"
                 name: env image helm overrides
+      - when: 
```

Comment:
isn't `setup-gke-cluster` creating a new cluster? Or am I misinterpreting this config?

Reply:
No, you're right! This was my bad, I thought you used this step but just realized you created a new step called `setup-shared-cluster`, where this code should be instead. 
Do you want me to add this code to the step you created and then commit it to your branch? Or would you rather wait until your branch is merged into the feature branch before we implement this addon?

Reply:
In that case, we wouldn't need these params, so I guess we would just have to incorporate the step that adds the CI source IP ranges to `HELM_VALUES`.